core/tests/Test.Microsoft.Identity.LabInfrastructure/LabServiceApi.cs

```diff
@@ -71,12 +71,22 @@ private LabResponse GetLabResponseFromAPI(UserQueryParameters query)
 
             queryDict.Add("external", query.IsExternalUser != null && (bool)(query.IsExternalUser) ? "true" : "false");
 
-            if (query.UserType == UserType.B2C)
+            if (query.B2CIdentityProvider == B2CIdentityProvider.Local)
             {
                 queryDict.Add("b2cProvider", "local");
             }
 
-            UriBuilder uriBuilder = new UriBuilder("http://api.msidlab.com/api/user");
+            if (query.B2CIdentityProvider == B2CIdentityProvider.Facebook)
+            {
+                queryDict.Add("b2cProvider", "facebook");
+            }
+
+            if (query.B2CIdentityProvider == B2CIdentityProvider.Google)
+            {
+                queryDict.Add("b2cProvider", "google");
+            }
+
+            UriBuilder uriBuilder = new UriBuilder("http://api.msidlab.com/api/userbeta");
```

Comment:
>UriBuilder uriBuilder = new UriBuilder("http://api.msidlab.com/api/userbeta"); [](start = 12, length = 78)

will the beta endpoint continue to be available? an ETA on this going to the non beta end-point?

Reply:
i thought it would be today, so hopefully monday. I wanted to run a release build to test some b2c samples. this is temporary.

---
In reply to: [234387151](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1405#discussion_r234387151) [](ancestors = 234387151)